Gems/ScriptCanvas/Code/Builder/ScriptCanvasBuilderWorker.cpp

```diff
@@ -306,11 +315,11 @@ namespace ScriptCanvasBuilder
         else
         {
             // force load all dependencies into memory
-            for (auto& dependency : m_processEditorAssetDependencies)
-            {
-                auto depAsset = AZ::Data::AssetManager::Instance().GetAsset(dependency.m_assetId, dependency.m_assetType, AZ::Data::AssetLoadBehavior::PreLoad);
-                depAsset.BlockUntilLoadComplete();
-            }
+//             for (auto& dependency : m_processEditorAssetDependencies)
```

Comment:
delete unused code?  we can find this block in history if we ever want it again.